extras/uwsgi/uwsgi.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=uWSGI Emperor
-
-[Service]
-ExecStart=/usr/bin/uwsgi --emperor /etc/uwsgi/apps-enabled --logto /var/log/uwsgi/access.log
-RuntimeDirectory=uwsgi
-Restart=always
-KillSignal=SIGQUIT
-Type=notify
-NotifyAccess=all
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
Ok I can choose the first option to make sure that uwsgi is starting after MongoDB. 
Indeed Ideascube does not use MongoDB, but I was sharing this unit with an other web app, I will take care of this in Ansiblecube